docs/3.0.0-beta.x/getting-started/deployment.md

```diff
@@ -0,0 +1,190 @@
+    </template>
+		<template #title>Azure</template>
+		<template #description>
+			Step by step guide for deploying on Azure web app
+		</template>
+	</InstallLink>
+</div>
+
+<div>
+	<InstallLink link="../deployment/digitalocean">
+		<template #icon>
+			<svg width="178" height="177" viewBox="0 0 178 177" xmlns="http://www.w3.org/2000/svg"><g fill="#fff" fill-rule="evenodd"><path d="M89 176.5v-34.2c36.2 0 64.3-35.9 50.4-74-5.1-14-16.4-25.3-30.5-30.4-38.1-13.8-74 14.2-74 50.4H.8C.8 30.6 56.6-14.4 117.1 4.5c26.4 8.3 47.5 29.3 55.7 55.7 18.9 60.5-26.1 116.3-83.8 116.3z" fill-rule="nonzero"></path><path d="M89.1 142.5H55v-34.1h34.1zM55 168.6H28.9v-26.1H55zM28.9 142.5H7v-21.9h21.9v21.9z"></path></g></svg>
+		</template>
+		<template #title>DigitalOcean</template>
+		<template #description>
+			Manual step by step guide for deploying on DigitalOcean droplets
+		</template>
+	</InstallLink>
+</div>
+
+<div>
+	<InstallLink link="../deployment/heroku">
+    <template #icon>
+    <svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 5.12 5.12" preserveAspectRatio="xMinYMin meet"><path d="M3.068 4.415V2.382s.132-.487-1.63.2C1.436 2.6 1.436.7 1.436.7L2.01.697v1.2s1.61-.635 1.61.48v2.026h-.555zm.328-2.986h-.6c.22-.27.42-.73.42-.73h.63s-.108.3-.44.73zm-1.95 2.982V3.254l.58.58-.58.58z" fill="#fff"/></svg>
+    </template>
+		<template #title>Heroku</template>
+		<template #description>
+			Step by step guide for deploying on Heroku
+		</template>
+	</InstallLink>
+</div>
+
+<div>
+	<InstallLink link="../deployment/proxy">
+    <template #icon>
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="-35.5 26 32 32" width="64" height="64"><path d="M-33.442 42.023v-7.637a.68.68 0 0 1 .385-.651l13.173-7.608c.237-.148.503-.178.74-.03l13.232 7.637a.71.71 0 0 1 .355.651V49.63a.71.71 0 0 1-.355.651l-11.367 6.57a56.27 56.27 0 0 1-1.806 1.036c-.266.148-.533.148-.8 0l-13.202-7.608c-.237-.148-.355-.326-.355-.622v-7.637z" fill="#fff"/><path d="M-24.118 39.18v8.9c0 1.006-.8 1.894-1.865 1.865-.65-.03-1.154-.296-1.5-.858-.178-.266-.237-.562-.237-.888V35.836c0-.83.503-1.42 1.154-1.687s1.302-.207 1.954 0c.622.178 1.095.562 1.5 1.036l7.874 9.443c.03.03.06.09.118.148v-9c0-.947.65-1.687 1.57-1.776 1.154-.148 1.924.68 2.042 1.54v12.6c0 .7-.326 1.214-.918 1.54-.444.237-.918.296-1.42.266a3.23 3.23 0 0 1-1.954-.829c-.296-.266-.503-.592-.77-.888l-7.49-8.97c0-.03-.03-.06-.06-.09z" fill="#3498DB"/></svg>
+    </template>
+		<template #title>Proxy</template>
+		<template #description>
+			Overview of various Proxy options and sample configurations
+		</template>
+	</InstallLink>
+</div>
+
+### Configuration
```

Comment:
Transform this in `##` and rename with `Application configuration` just to precise that is for the Strapi app.